### ISSUE TITLE
highscores de concursos

### ISSUE THREAD

**Issue by armadaazteca** [Armada-Azteca/ArmadaAztecaServer#220]
Que en la pagina así como salen los ranks de los skills, salgan los ranks de los diferentes concursos de armada.

Para esto hay que considerar los siguientes cambios/scripts:

- [ ] Cambiar el comando de win, que el GM pueda especificar que concurso fue el que gano a ver aquí si se te ocurre algo @Ponid para que sea fácil para el gm ( mas abajo explico como creo que se puede hacer), 
- [ ] Hacer la base de datos de manera que no ocupe mucho espacio 
- [ ] Que los concursos automáticos suban el contador por si solos.
- [ ] Agregar un modulo en la pagina que muestre el top 10

**Comment by armadaazteca** [comment]
para el comando de win, estaba pensando algo asi como:
`!ganador "Aquiles, pla`   (Aquí el script va a checar si hay algún nombre de concurso que empiece con "**pla**" si solo hay uno, entonces que le asigne el nombre completo del concurso encontrado, si hay mas de uno entonces que le salga error al GM con la lista de concursos encontrados que empiezan con pla)

Con esa opción creo que no seria tan tedioso para el gm dar las wins especificas por el concurso y ademas si escribe algo que no existe como un concurso, entonces que salga error también. Es creo yo, la forma mas facil para el gm, pero hay que ver como scriptearla. Tengo 1 tabla con la lista de concursos y su nombre exacto y la pos inicial para el gm y creo que hay dos o tres especiales que no estan aqui, pero ps los tendriamos que checar que pedo esos ya de 1 por 1.

**Comment by Aron154** [comment]
yo digo que mejor que no importe el concurso si no el que mas gana los concursos te ahorrarias tiempo y seria casi igual

**Comment by Ponid** [comment]
Estoy de acuerdo con @Aron154, algo así como lo que pusiste el día del niño 2013, se veía muy bien. Y al top 3 por semana que se le de una win especial o algo así (una sala de premios que solo lo pueda mandar el GOD, ya que abría premios mas caros que los normales) esto aria que participaran más en los concursos.

**Comment by armadaazteca** [comment]
ok, pueodo poner entonces que solo los automáticos te cuenten especial. Y los que activa el GM cuenten solo de manera global.
